src/chart/animator/planner.cpp

```diff
@@ -222,7 +230,10 @@ void Planner::calcNeeded()
 	        actual->getStyle())
 	        .isNeeded();
 
-	animNeeded[SectionId::title] = srcOpt->title != trgOpt->title;
+	animNeeded[SectionId::title] =
```

Comment:
We should handle title, subtitle and footer animation separately.

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/349/commits/be8842bd1203c5601a3550b0a63eeb63d9442f3b